{
  "done": [
    "librttopo",
    "libspatialite",
    "gdal",
    "postgis",
    "r-rgeos",
    "basemap",
    "shapely",
    "cartopy"
  ],
  "in-pr": [
    "qgis",
    "omniscidb",
    "pygeos"
  ],
  "awaiting-pr": [],
  "not-solvable": [
    "r-terra",
    "gmt"
  ],
  "awaiting-parents": [
    "r-exactextractr",
    "pyogrio"
  ],
  "bot-error": [],
  "_feedstock_status": {
    "basemap": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/basemap-feedstock/pull/86",
      "pr_status": "unstable"
    },
    "cartopy": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/cartopy-feedstock/pull/132",
      "pr_status": "unknown"
    },
    "gdal": {
      "num_descendants": 9,
      "immediate_children": [
        "gmt",
        "omniscidb",
        "postgis",
        "pyogrio",
        "qgis",
        "r-exactextractr",
        "r-terra",
        "rsgislib",
        "spdlib"
      ],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/gdal-feedstock/pull/577",
      "pr_status": "unknown"
    },
    "gmt": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "not solvable (<a href=\"https://github.com/regro/autotick-bot/actions/runs/1506438254\">bot CI job</a>): master: ['linux_64_: Encountered problems while solving:\\n  - package graphicsmagick-1.3.30-h2cf0850_0 requires gnuplot, but none of the providers can be installed\\n', 'osx_64_: Encountered problems while solving:\\n  - nothing provides qt 4.8.* needed by gnuplot-5.0.5-0\\n']"
    },
    "libspatialite": {
      "num_descendants": 11,
      "immediate_children": [
        "gdal",
        "libgdal",
        "qgis"
      ],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/libspatialite-feedstock/pull/80",
      "pr_status": "unknown"
    },
    "postgis": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/postgis-feedstock/pull/70",
      "pr_status": "unknown"
    },
    "qgis": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/qgis-feedstock/pull/216",
      "pr_status": "unstable"
    },
    "shapely": {
      "num_descendants": 1,
      "immediate_children": [
        "cartopy"
      ],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/shapely-feedstock/pull/93",
      "pr_status": "unknown"
    },
    "r-rgeos": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/r-rgeos-feedstock/pull/30",
      "pr_status": "unknown"
    },
    "pygeos": {
      "num_descendants": 1,
      "immediate_children": [
        "pyogrio"
      ],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/pygeos-feedstock/pull/48",
      "pr_status": "unstable"
    },
    "r-exactextractr": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": ""
    },
    "omniscidb": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/omniscidb-feedstock/pull/67",
      "pr_status": "clean"
    },
    "pyogrio": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": ""
    },
    "librttopo": {
      "num_descendants": 12,
      "immediate_children": [
        "libspatialite"
      ],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/librttopo-feedstock/pull/11",
      "pr_status": "unknown"
    },
    "r-terra": {
      "num_descendants": 1,
      "immediate_children": [
        "r-exactextractr"
      ],
      "pre_pr_migrator_status": "not solvable (<a href=\"https://github.com/regro/autotick-bot/actions/runs/1506438254\">bot CI job</a>): master: ['linux_64_r_base4.0: Encountered problems while solving:\\n  - package libgdal-3.3.3-h18e3bf0_2 requires geos >=3.10.0,<3.10.1.0a0, but none of the providers can be installed\\n', 'linux_64_r_base4.1: Encountered problems while solving:\\n  - package libgdal-3.3.3-h18e3bf0_2 requires geos >=3.10.0,<3.10.1.0a0, but none of the providers can be installed\\n', 'linux_aarch64_r_base4.0: Encountered problems while solving:\\n  - package libgdal-3.3.3-h2f695e7_2 requires geos >=3.10.0,<3.10.1.0a0, but none of the providers can be installed\\n', 'linux_aarch64_r_base4.1: Encountered problems while solving:\\n  - package libgdal-3.3.3-h2f695e7_2 requires geos >=3.10.0,<3.10.1.0a0, but none of the providers can be installed\\n', 'linux_ppc64le_r_base4.0: Encountered problems while solving:\\n  - package libgdal-3.3.3-h111df2f_2 requires geos >=3.10.0,<3.10.1.0a0, but none of the providers can be installed\\n', 'linux_ppc64le_r_base4.1: Encountered problems while solving:\\n  - package libgdal-3.3.3-h111df2f_2 requires geos >=3.10.0,<3.10.1.0a0, but none of the providers can be installed\\n', 'osx_64_r_base4.0: Encountered problems while solving:\\n  - package libgdal-3.3.3-h19466b5_2 requires geos >=3.10.0,<3.10.1.0a0, but none of the providers can be installed\\n', 'osx_64_r_base4.1: Encountered problems while solving:\\n  - package libgdal-3.3.3-h19466b5_2 requires geos >=3.10.0,<3.10.1.0a0, but none of the providers can be installed\\n']"
    }
  }
}